### PR TITLE
Update/slot fill docs

### PR DESCRIPTION
**Pull request by ryanwelcher**
## Description
This PR addresses incorrect image path issues when SlotFill is rendered in the handbook.

Example: https://developer.wordpress.org/block-editor/developers/slotfills/plugin-block-settings-menu-item/

### CODE REVIEW

**Comment by ryanwelcher** [comment]
Closes #15744

**Comment by ryanwelcher** [comment]
@noisysocks looks like my image paths we incorrect. I've pushed a change to this branch to address that.

File: docs/designers-developers/developers/slotfills/plugin-block-settings-menu-item.md [hunk 1]
```diff
@@ -23,5 +23,5 @@ registerPlugin( 'block-settings-menu-group-test', { render: PluginBlockSettingsM
 
 ## Location
 
-![Alt text](/docs/designers-developers/assets/plugin-block-settings-menu-item-screenshot.png?raw=true "PluginBlockSettingsMenuItem Location")
+![Location](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/assets/plugin-block-settings-menu-item-screenshot.png?raw=true "PluginBlockSettingsMenuItem Location")
```
Comment (nosolosw) [inline docs/designers-developers/developers/slotfills/plugin-block-settings-menu-item.md]:
```suggestion
![Location](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/assets/plugin-block-settings-menu-item-screenshot.png?raw=true)
```

**Review by oandregal — APPROVED** [review]
Thanks for the quick turnaround.

**Comment by nosolosw** [comment]
@ryanwelcher the [docs](https://developer.wordpress.org/block-editor/contributors/document/) need updating. Would you have the availability to prepare a PR to add a section on how to add images to the handbook?

**Comment by ryanwelcher** [comment]
@nosolosw I created an issue for that #15811

**Pull request merged**